SnapshotTests/DefaulController/DefaulController.swift

```diff
@@ -0,0 +1,15 @@
+//  Copyright © 2017 rtysiachnik. All rights reserved.
```

Comment:
Change everywhere in this PR to - `//  Copyright © 2017 Oath. All rights reserved.`

Reply:
Got it.